packages/components/src/TextArea/TextArea.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
```

Comment:
I think `TextArea` component can be under the `Form`directory same as [antd](https://ant.design/components/input/#components-input-demo-textarea). How do you think?

Reply:
Yep, moved to Form.